packaging/packager_deb.sh

```diff
@@ -5,46 +5,61 @@ readonly distro="${DISTRO:-ubuntu}"
 readonly distro_version="${VERSION:-xenial}"
 readonly distro_root="${ROOT:-$HOME}"
 
-# deb dependencies
+echo "------------- deb dependencies -------------"
 export DEBIAN_FRONTEND=noninteractive
 apt-get update
-# python3
-apt-get -y install python3 python3-coverage
-# python-imaging
-apt-get -y install python3-pil
-# pip dependencies (for dependencies not available as DEB)
+echo "------------- python3 install -------------"
```

Comment:
It is meant to be "python3" or "python3 install"? Please unify all section names across all packagers